test/core/test/injector-mock.test.ts

```diff
@@ -67,7 +67,6 @@ test('correctly mocks namespaced', () => {
   `)).toMatchInlineSnapshot(`
     "const { vi } = await import('vitest')
     vi.mock('../src/add', () => {})
-    const AddModule = await import('../src/add')
-    const { default: add } = AddModule"
```

Comment:
There are no tests that check if properties are accessed correctly. Maybe we can take them from Vite? This function originated there

Reply:
It doesn't look like we need to do this. The current behavior also works fine

Reply:
No, we _need_ to have tests.

Reply:
I checked in vite. Looks like we don't need this line `const { default: add } = AddModule"
`
```ts
  expect(await ssrTransformSimpleCode(`
    import { vi } from 'vitest'
    import add, * as AddModule from '../src/add'
    add();
    vi.mock('../src/add', () => {})`))
.toMatchInlineSnapshot(`
    "const __vite_ssr_import_0__ = await __vite_ssr_import__("vitest", {"importedNames":["vi"]});
    const __vite_ssr_import_1__ = await __vite_ssr_import__("../src/add", {"importedNames":["default"]});

      
      
      __vite_ssr_import_1__.default();
      __vite_ssr_import_0__.vi.mock('../src/add', () => {})"
  `)
```

Reply:
Then add these tests in this pr